src/services/groups-service/groups-service.ts

```diff
@@ -37,20 +37,34 @@ export class GroupsService<T extends GroupResource = GroupResource> extends Tras
         super(serverApi, "groups", actions);
     }
 
-    contents(uuid: string, args: ContentsArguments = {}): Promise<ListResults<GroupContentsResource>> {
+    async contents(uuid: string, args: ContentsArguments = {}): Promise<ListResults<GroupContentsResource>> {
         const { filters, order, ...other } = args;
         const params = {
             ...other,
             filters: filters ? `[${filters}]` : undefined,
             order: order ? order : undefined
         };
-        return CommonResourceService.defaultResponse(
+
+        const response = await CommonResourceService.defaultResponse(
             this.serverApi
                 .get(this.resourceType + `${uuid}/contents`, {
                     params: CommonResourceService.mapKeys(_.snakeCase)(params)
                 }),
-            this.actions
+            this.actions, 
+            false
         );
+        const { items, ...res } = response;
+        const mappedItems = items.map((item: any) => {
```

Comment:
Avoid `any`. I think item type is inferred from items array and there is no need to add type info here.